doc/install/frontend-development.md

```diff
@@ -157,3 +157,18 @@ or rebuild node-sass:
 ```
 npm rebuild node-sass
 ```
+
+**docker container can't build or start**
+
+the following error can appears when the host user (you) can use directly  docker and when a file is owned by root only:
```

Comment:
**T**he following… (missing cap)

Reply:
"can appear" also, and the next phrase is not very explicit, I would say "when docker is not run as root"